src/plugin/index-more/src/java/org/apache/nutch/indexer/more/MoreIndexingFilter.java

```diff
@@ -324,16 +358,19 @@ public Configuration getConf() {
 
   private void readConfiguration() throws IOException {
     LOG.info("Reading content type mappings from file contenttype-mapping.txt");
-    BufferedReader reader = new BufferedReader(
-        conf.getConfResourceAsReader("contenttype-mapping.txt"));
-    String line;
-    String parts[];
-    boolean formatWarningShown = false;
+    try(BufferedReader reader = new BufferedReader(
```

Comment:
nit: Not sure why we need this `try` block? If a different exception gets thrown then it should be added to the method signature.

Reply:
it will auto-close the reader (stream), which was not done here at all.